### DISCUSSION TITLE
How to show the sum of column values

### DISCUSSION THREAD

**Discussion by ali-d-coded** [Power-Components/livewire-powergrid#329] [Q&A] [answered]
How can I show the sum of a particular column's values on the top of that column?

**Comment by dansysanalyst** [comment, marked as answer]
Please, make use of the documentation: 📚 https://livewire-powergrid.com

https://livewire-powergrid.com/#/table/cell-actions-buttons?id=withsumstring-label-sum-bool-header-false-bool-footer-false

**Reply by ali-d-coded** [reply]
This was extremely helpful!! Thank you very much.

**Comment by dansysanalyst** [comment]
Glad to hear!

________
⭐ Enjoying PowerGrid? Please consider Starring the repository!
📢 Share PowerGrid with your dev-community, friends and colleagues: Tweet, write a blog, record review video. 
👥 PowerGrid is open-source, entirely developed and maintained by the author and the contributors.

**Reply by ali-d-coded** [reply]
already did!
